src/blog/jwt-identification-and-hot-streams/index.md

```diff
@@ -0,0 +1,173 @@
+# AnyCable PRO:<br>JWT identification and "hot streams"
+
+September 27, 2021
+{date}
+
+When we started collecting applications for the [AnyCable PRO][pro] early access program back in July, we asked respondents to share how they use _cables_ and which features they would like us to provide out of the box. Analyzing these responses led to the development of two new PRO features, [JWT identification][jwt-id-docs] and [signed streams support][signed-streams-docs]. They might seem unrelated to each other at first glance, but when used together, they have the potential to bring about a huge performance boost. Continue reading to learn more!
+{intro}
+
+<div class="divider"></div>
+
+## Dealing with authentication
+
+Amongst all the requested features, one of the leading requests was a "token-based authentication". Most Action Cable applications rely on cookies as an authentication mechanism (either directly or via sessions). The main benefit of cookie-based authentication is simplicity—it just works. However, there are some drawbacks:
```

Comment:
Again, I think we should explain to newcomers, why we are talking about Action Cable here. I'd like to suggest something like that (but it may be not technically correct so please help me to improve):

AnyCable reckons upon the Action Cable API (although it can work outside of Rails) and most Action Cable applications rely on cookies as an authentication mechanism (either directly or via sessions).